pkgs/development/python-modules/pgzero/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python3Packages
+}:
+
+buildPythonPackage rec {
+  pname = "pgzero";
+  version = "1.2.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-jK3AIPAoy6w+DL07uTEaHARfHe7ax5F/9DP5hsOOYQY=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    enum34
+    numpy
+    pygame
+  ];
+
```

Comment:
We should execute the tests https://github.com/lordmauve/pgzero/tree/master/test

Reply:
tests related to sound are not working so i have deleted the files that contains these tests

https://github.com/lordmauve/pgzero/issues/305